### PR TITLE
Fixed docker.containers

### LINKED ISSUE #68: Check docker.containers
Ensure that all content is working. (all cases, input errors etc.)

### PR DESCRIPTION
**Pull request by tudorlesan**
Signed-off-by: Lesan Tudor <tudor-andrei.lesan@hp.com>
fixes #68

### CODE REVIEW

**Comment by tudorlesan** [comment]
Added changes to docker.containers to this branch to avoid to many changes on just one branch. @oritstone @SamMarkowitz could you review please?

**Comment by tudorlesan** [comment]
fixes #68

File: content/org/openscore/slang/base/network/verify_app_is_up.sl [hunk 1]
```diff
@@ -46,7 +46,7 @@ operation:
             if code == 200 :
               return_result = True
   outputs:
-    - error_message: "'Application is not Up , after ' + count + ' attempts to ping .'"
+    - error_message: "'Application is not Up , after ' + str(count) + ' attempts to ping .'"
   results:
```
Comment (SamMarkowitz) [inline content/org/openscore/slang/base/network/verify_app_is_up.sl]: Change "Up" to lowercase "up" and remove the comma (make sure there is only one space left).

File: content/org/openscore/slang/docker/containers/get_container_ip.sl [hunk 2]
```diff
@@ -63,8 +63,8 @@ operation:
       className: org.openscore.content.ssh.actions.SSHShellCommandAction
       methodName: runSshShellCommand
   outputs:
-    - db_IP: returnResult[:-1]
-    - error_message: STDERR if returnCode == '0' else returnResult
+    - container_ip: returnResult[:-1]
```
Comment (SamMarkowitz) [inline content/org/openscore/slang/docker/containers/get_container_ip.sl]: Change doc comment to reflect new name.

**Pull request merged**